.travis.yml

```diff
@@ -5,8 +5,6 @@ python:
   - 3.3
   - 3.4
 env:
-  matrix:
-    - DJANGO=django==1.7.3
```

Comment:
No, it's best to test against multiple major releases of Django.

See example here:

https://travis-ci.org/etianen/django-s3-storage